Shared/UI/Views/MessageContentView/MessageContentView.swift

```diff
@@ -16,8 +16,8 @@ import StreamChat
 class MessageContentView: View {
 
     #if IOS
-    var handleTappedMessage: ((MessageSequenceItem) -> Void)?
-    var handleEditMessage: ((MessageSequenceItem) -> Void)?
+    var handleTappedMessage: ((ConversationID, MessageId) -> Void)?
+    var handleEditMessage: ((ConversationID, MessageId) -> Void)?
```

Comment:
There are two enum cases of message sequence items now (Message, LoadMore) so I'm just returning the relevant message info so handlers don't have to write a switch case.